graphql/server.go

```diff
@@ -283,7 +283,7 @@ func (c *conn) handleMutate(in *inEnvelope) error {
 	}
 
 	initial := true
-	e := Executor{}
+	e := BatchExecutor{}
```

Comment:
I think you'll need to call `NewBatchExecutor()` here, otherwise the scheduler won't be set?  (Old executor doesn't have this problem because it's only field is a mutex, which has a sensible zero value)